README.md

```diff
@@ -130,10 +130,158 @@ From an exixting micromamba environment, you can now install manually `py4cast`
 pip install --editable .
 ```
 
+### Build docker image
+
+To build the docker image please use the `build-docker.sh` script.
```

Comment:
I don't see the build-podman.sh script in your PR, and the docker one is named docker-build.sh @A669015 . Maybe git add and harmonize names ?

Reply:
My bad, I didn't use the proper name in the doc.
There is one unified script, for docker and podman (a parameter of the script allows to choose the format), so maybe the script should be renamed. oci-image-build.sh ? Any other idea ?

Reply:
Maybe the name of the option is not very xell chosen to. It could be:
`bash ./oci-image-build.sh --builder docker` and `bash ./oci-image-build.sh --builder podman`

Reply:
**bash ./oci-image-build.sh --runtime docker**

https://kubernetes.io/docs/setup/production-environment/container-runtimes/

https://www.wiz.io/academy/container-runtimes